src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo,
+  selectedTodo: Todo | null,
+  changeSelectedTodo: (value: Todo) => void,
+};
+
+export const TodoItem: React.FC<Props> = ({
+  selectedTodo,
+  todo,
+  changeSelectedTodo,
+}) => {
+  const { id, title, completed } = todo;
+  const isTodoSelected = id === selectedTodo?.id;
+
+  return (
+    <tr data-cy="todo" className="">
+      <td className="is-vcentered">
+        {id}
+      </td>
+
+      <td className="is-vcentered">
+        {completed && (
+          <span className="icon" data-cy="iconCompleted">
+            <i className="fas fa-check" />
+          </span>
+        )}
+      </td>
+
+      <td className="is-vcentered is-expanded">
+        <p
+          className={completed
+            ? 'has-text-success'
+            : 'has-text-danger'}
```

Comment:
Use `classnames`

Reply:
Is it a good idea to use a classnames in this case? From my perspective, in this situation when we are choosing 1 class of 2 it is better to use this operator to avoid duplications.

Reply:
I guess that yes. It will be also easy to read. But also in the future, if you need to add 1 more class, you will refactor your code anyway using `classnames`, so why not use it immediately